app/src/main/res/resource/values-de/strings.xml

```diff
@@ -190,7 +192,7 @@
     <string name="action_about">Über</string>
```

Comment:
https://github.com/WangDaYeeeeee/Mysplash/pull/44/files#diff-b0f2ab15ff4a50a4f9a5584648bdcf1dR186
"Following" is likely a button/list header here. "Folgende" is clearly wrong. Just use the English word: "Followers". Or "Deine Followers"?